### PR TITLE
[qa] Fixed failing commit checks for co-authored commits #167

### LINKED ISSUE #167: [qa] Failing commit checks for co-authored commits
QA checks have started to fail for commits that are co-authored by one or more users.
For example:
1. https://github.com/openwisp/openwisp-controller/runs/1574632899
2. https://github.com/openwisp/ansible-openwisp2/runs/1546631993

### Expected
These checks should not have failed.

### PR DESCRIPTION
**Pull request by devkapilbansal**
Fixes #167

### CODE REVIEW

**Comment by devkapilbansal** [comment]
@nemesisdesign when merging pull request through github interface, a carriage return is added at end of commit message(long description) due to which checks fails.

File: openwisp_utils/qa.py [hunk 1]
```diff
@@ -228,7 +228,7 @@ def _find_issue_mentions(message):
      - count of mentions performed correctly
        (using one of the common github keywords)
     """
-    words = message.split(' ')
+    words = message.split()
```
Comment (devkapilbansal) [inline openwisp_utils/qa.py:231]: By removing quotes from here, words will be splitted for any white space (carriage return, new line etc too)

**Comment by devkapilbansal** [comment]
Will fix tests soon

**Review by nemesisdesign — CHANGES_REQUESTED** [review]
can you please add a test for this case?
I had tried it but couldn't replicate it, but I didn't try adding the carriage return in the test text.
See: https://github.com/openwisp/openwisp-utils/blob/42b2500774e9adb23af53b7415f9229a34a86c38/tests/test_project/tests/test_qa.py#L130-L181

File: tests/test_project/tests/test_qa.py [hunk 2]
```diff
@@ -118,6 +118,13 @@ def test_qa_call_check_commit_message_pass(self):
                 '[qa] Improved Y #20\n\n'
                 'Simulation of a special unplanned case\n\n#noqa',
             ],
+            [
+                'commitcheck',
+                '--quiet',
+                '--message',
+                '[fix] Fixed extensibility of openwisp-users and added sample_users test app #377\n\n'
+                'Closes #377\r\n\r\nCo-authored-by: Ajay Tripathi <ajay39in@gmail.com>',
```
Comment (devkapilbansal) [inline tests/test_project/tests/test_qa.py:126]: This is the commit message I copied from openwisp-controller for which tests are failing

File: openwisp_utils/qa.py (hunk 1 above)
Comment (nemesisdesign) [reply openwisp_utils/qa.py:231]: I hadn't noticed this before, thanks!

**Review by nemesisdesign — APPROVED** [review]
Thanks @devkapilbansal

**Pull request merged**